src/main/java/com/wolox/training/controllers/BookController.java

```diff
@@ -20,13 +25,11 @@
 
 @RestController
 @RequestMapping("/api/books")
+@Api(tags = "Books")
 public class BookController {
```

Comment:
Add the @ApiParam annotation to specify and describe the params needed.

Reply:
done!